src/styles/brackets.less

```diff
@@ -1893,6 +1893,7 @@ textarea.exclusions-editor {
     border-radius: 0 0 4px 4px;
     box-shadow: 0 5px 10px @bc-shadow;
     opacity: 0;
+    overflow-wrap: break-word;
```

Comment:
This is going to add line break instead of ...
Is this expected?
It's going to impact normal quick open (files from current project). Is that fine?
If we have small window size, it's behaving weirdly. Anyways that's nothing to do with this change but just letting you know if you want to have a look? (existing issue but it can affect more in this feature)

Reply:
Yeah. Wrapping is expected. Wrapping was decided to be the expected behavior for this issue. 
I will take a look on the other issues on window resize and see if I can fix it.